src/main/java/org/ngafid/CalculateProximity.java

```diff
@@ -2,46 +2,34 @@
 
 import org.ngafid.Database;
 import org.ngafid.events.Event;
+import org.ngafid.events.RateOfClosure;
 import org.ngafid.flights.DoubleTimeSeries;
 import org.ngafid.flights.Flight;
 import org.ngafid.flights.StringTimeSeries;
-
 import java.math.BigDecimal;
 import java.sql.Connection;
 import java.sql.PreparedStatement;
 import java.sql.ResultSet;
 import java.sql.SQLException;
-
 import java.time.Duration;
 import java.time.Instant;
-
 import java.time.LocalDateTime;
 import java.time.OffsetDateTime;
 import java.time.ZonedDateTime;
 import java.time.ZoneOffset;
 import java.time.format.DateTimeFormatter;
-
-
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.HashMap;
-import java.util.Map;
-import java.util.LinkedHashSet;
+import java.util.*;
```

Comment:
Could you be specific here with the imports? It's not good form to do import * (it's slower and less specific).